layouts/shortcodes/docs/instrumentation/lang-index-head.md

```diff
@@ -0,0 +1,35 @@
+{{ $data := index $.Site.Data.instrumentation.languages (.Get 0) }}
```

Comment:
Maybe this should be renamed to `lang-index-intro.md`? WDYT @svrnm?

Reply:
Yeah ... I prefer "intro". The term "head" makes me think that it includes content in `<head>`. Let me make the change.